gradle-jdks/src/main/java/com/palantir/gradle/jdks/GradleJdksConfigs.java

```diff
@@ -65,6 +66,12 @@ public final void setIncludeVersionsFromCli(List<String> versions) {
         getIncludeJavaMajorVersions().set(versions);
     }
 
+    @Input
+    @Option(
+            option = "onlyForCurrentOsArch",
+            description = "Generates the configuration directories only for the current Os & Arch.")
+    public abstract Property<Boolean> getIncludeOnlyCurrentOsArch();
```

Comment:
example of usage in the gradle-plugin-testing invocation: https://github.com/palantir/gradle-plugin-testing/compare/cr/gradlew-gradleInvoker?expand=1#diff-4b13fa970a8965fd58bc7f42781597ec77437fba2dc9e021175c30ba85e5bf95R46